jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  // Restores all mock / spied-on implementations before each test
+  restoreMocks: true,
```

Comment:
This restores any `jest.spyOn` calls after each test. Open to whatever you'd prefer here, but my experience in the past with Jest has been:
* Re-creating the mock before each test isn't a big deal most of the time
* Forgetting to reset the mock after a test can lead to confusing test failures

Reply:
Sounds good to me. I think I hacked together something like this in router.test.js.